lib/spack/spack/test/build_systems.py

```diff
@@ -59,6 +59,7 @@ def test_affirmative_make_check(directory, config, mock_packages):
     'directory',
     glob.iglob(os.path.join(DATA_PATH, 'make', 'negative', '*'))
 )
+@pytest.mark.regression('9067')
```

Comment:
Does this regression actually make sense here? I'm confused why only certain negative tests failed and not all of them for newer versions of Make.

Reply:
I marked the regression (even though I didn't add a new test or modified an existing one) to leave information on a related issue.

> I'm confused why only certain negative tests failed and not all of them for newer versions of Make.

I am not :smile: There are 3 tests, where one checks that Spack ignores directories without a Makefile. That one passed because `_has_make_target` returned a result before invoking `make`.

Reply:
To be clearer, the test that succeeds returns a value here:

https://github.com/spack/spack/blob/f7068facee33b31796b34398b9b0d67d16297b34/lib/spack/spack/package.py#L1153-L1159